vm/Cargo.toml

```diff
@@ -123,6 +123,7 @@ openssl = { version = "0.10.32", features = ["vendored"], optional = true }
 openssl-sys = { version = "0.9", optional = true }
 openssl-probe = { version = "0.1", optional = true }
 which = "4.0"
+filepath = "0.1.1"
```

Comment:
It looks like this only supports windows/mac/linux, so it might be good to put the feature gate behind `[target.'cfg(any(target_os = "linux", target_os = "macos", windows))'.dependencies]`

Reply:
Thanks and fixed!